docs/source/debugging-optimizing/numactl-utility.md

```diff
@@ -16,14 +16,20 @@ limitations under the License.
 
 # Using the numactl Utility to Control Resource Utilization with the DeepSparse Engine
 
-The DeepSparse Engine works best when run on a single socket and with hyper-threading disabled. One standard way of controlling compute/memory resources when running processes is to use the **numactl** utility. **numactl** can be used when multiple processes need to run on the same hardware but require their own CPU/memory resources to run optimally.
+The DeepSparse Engine supports multiple-socket systems. With hyper-threading disabled, models with larger batch sizes are likely to see improvement. One standard way of controlling compute/memory resources when running processes is to use the **numactl** utility. **numactl** can be used when multiple processes need to run on the same hardware but require their own CPU/memory resources to run optimally.
```

Comment:
I think "With hyper-threading disabled," should just be removed from this sentence and then add a sentence at the end that says "The engine has better performance when hyper-threading is disabled".  This is true regardless of the number of sockets.